data_structures/linked_list/__init__.py

```diff
@@ -25,6 +25,26 @@ def remove(self):
     def is_empty(self):
         return self.head is None
 
+    def __str__(self):
+        """
+        >>> linked_list = LinkedList()
+        >>> linked_list.add(23)
+        >>> linked_list.add(14)
+        >>> linked_list.add(9)
+        >>> print(linked_list)
+        9 --> 14 --> 23 -->
+        """
+        if not self.is_empty:
+            return "Linked List is empty."
+        else:
+            iterate = self.head
+            iteam_str = ""
+            while iterate:
+                iteam_str += f"{iterate.item} --> "
+                iterate = iterate.next
+
+            return iteam_str
```

Comment:
```suggestion
            return iteam_str.strip()
```
The problem here is that the pre-commit just trims the whitespaces so the doctests fail

```python
>>> str(linked_list).strip() == str(linked_list)
False

>>> str(linked_list).strip() != str(linked_list)
True
```